src/old-events/quantum-computing-lecture.ht

```diff
@@ -1,9 +0,0 @@
-<div class="article">
```

Comment:
Do you know if these two `/src/old-events/` files are preserved anywhere? Can't seem to find any new md files with their contents.

Reply:
I 100% missed copying these files over! Good catch!

After code review is finished I'll bring these in, make the other changes listed above, and resubmit the PR!

Thansk!

Reply:
If you want, you can also just update this pull request by pushing changes to the same branch on your fork.

Reply:
Oh! **I had no idea you could do that! 😅**
Thanks!

Reply:
Ahaha, yeah, it's a super convenient part of the github workflow. Resubmitting PRs seems... really inconvenient.

On that topic, I should probably write up a list of GitHub workflow tips/tricks when I get the time to go under the contributing docs. One of the things that's made possible by this specific feature is that you can have 'WIP' pull requests, which are open so they can be commented on but are still very much a work in progress. Makes it much easier to incorporate review changes early on in feature development.

Reply:
That would be awesome! 

I've been using github / git for my personal projects for a few years but I've never actually contributed to an OS repo so I'm not super knowledgeable on the PR process / githubs features surrounding it!

Reply:
@MJJBennett addressed this and added events in [this commit](https://github.com/ufvcsa/ufvcsa.github.io/pull/16/commits/07a541e66e7b18366aa6bda6d7cf238a01e11377)